client/src/components/restaurant/restaurant-register.js

```diff
@@ -0,0 +1,224 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button, Form, Container, Alert,
+} from 'react-bootstrap';
+import '../../styles/restaurant-registration.css';
+import Api from '../../services/api';
+import MyBadge from '../shared/my-batch';
+
+class RestaurantRegistration extends Component {
+  constructor(props) {
+    super(props);
+    this.endpoint = '';
+    this.state = {
+      name: '',
+      email: '',
+      address: '',
+      time_from: '09:00',
+      time_to: '22:00',
+      description: '',
+      tables: 3,
+      isRegistered: false,
+    };
+    this.userID = localStorage.getItem('userID');
```

Comment:
Your code depends on low level storage. If we will change localstorage to other storage system, we 100% will forget to change this line and other similar

Reply:
This issue still relevant. You can use method `Auth.getPersonId()` https://github.com/Social-projects-Rivne/Lunch_Time/blob/dev/client/src/services/auth.js#L34

https://rv-054java.slack.com/archives/CUKLK9279/p1588940904017600

There are 2 places in your code, where you use `localstorage.getItem("key")`